pandas/tests/indexing/test_loc.py

```diff
@@ -107,25 +116,31 @@ def test_loc_getitem_label_list(self, key, typs):
             [[(1, 3), (1, 4), (2, 5)], ["multi"], 0],
         ],
     )
-    def test_loc_getitem_label_list_with_missing(self, key, typs, axes):
-        self.check_result("loc", key, typs=typs, axes=axes, fails=KeyError)
-
-    def test_loc_getitem_label_list_fails(self):
+    @pytest.mark.parametrize("kind", ["series", "frame"])
```

Comment:
fixture for the kind?

Reply:
Don't think it's entirely worth it here. This is just used to combine with another string to fetch a fixture.